website/docs/reference/root-dir.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Project root directory
+sidebar_position: 5
+---
+
+## Usage
+
+Scala CLI needs a root directory:
+  - to write mapped sources
+  - to write class files
+  - for Bloop
+
+## Setting root directory
+
+First of all Scala CLI checks every passed input (in the same order in which inputs were passed) for the `project.settings.scala` file:
+- If the `project.settings.scala` file is passed explicitly as a **source**, Scala CLI sets its parent directory as the root directory.
+- If the input is a **directory**, Scala CLI looks for the `project.settings.scala` inside this directory. If the file is found, Scala CLI sets the passed directory as the root directory.
+
+If more than one `project.settings.scala` file is found, Scala CLI uses only **the first one** to set the root directory and raises **warning** saying which one was used.
+
+If no `project.settings.scala` files are found, Scala CLI sets the root directory based on the first file/directory input:
+- If the input is a **directory**, it is set as the root directory. 
+- If the input is a **file**, Scala CLI sets its parent directory as the root directory. 
+
+If more then one file/directory input has ben passed Scala CLI raises the warning saying which directory has been set as the project root directory.
+
+If no `project.settings.scala` files are found and no file/directory inputs have ben passed, Scala CLI sets **cwd** as the project root directory.
```

Comment:
I think `cwd` isn't known widely enough to be used in our docs, let's state this more descriptively:
```suggestion
If no `project.settings.scala` files are found and no file/directory inputs have ben passed, Scala CLI sets the current working directory (where Scala CLI was invoked from) as the project root directory.
```